### PR TITLE
Update whitefox to use the K-Type teensy workaround

### PR DESCRIPTION
**Pull request by belak**
This also adds support for loading boards from a common location, so this falls back to `drivers/boards` if the board doesn't exist in the keyboard directories.

### CODE REVIEW

File: keyboards/whitefox/rules.mk [hunk 1]
```diff
@@ -52,7 +52,9 @@ ARMV = 7
 # 0x00000000-0x00001000 area is occupied by bootlaoder.*/
 # The CORTEX_VTOR... is needed only for MCHCK/Infinity KB
 #OPT_DEFS = -DCORTEX_VTOR_INIT=0x00001000
-OPT_DEFS = 
+OPT_DEFS =
+
+DFU_ARGS = -d 1c11:b007
```
Comment (belak) [inline keyboards/whitefox/rules.mk]: This is a somewhat unrelated QoL change. All I:C keyboards seem to use this when flashing... it shows up as two devices so the default dfu-util will bail without this.

**Comment by jackhumbert** [comment]
Perfect - thanks!
